### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by poornima-alugubelly**

### CODE REVIEW

**Review by Amandeep-sinha — CHANGES_REQUESTED** [review]
In Alert.html :- 

> I think you should print the Alert names in the respective order (line no 94 - 95)

> At line no.- 104 - I guess the class name will be "far" or let me know if I'm wrong

**Comment by poornima-alugubelly** [comment]
> 

Hey thank u so much for reply!
I was going to change the names but forgot thanks for pointing it out!
Yes it is fas , the far one is pro one

File: components/Alert/Alert.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.alert-icon {
+  margin-right: 0.75rem;
+}
+
+.alert-primary {
+  color: #7eafe4;
```
Comment (srishti-maurya) [inline components/Alert/Alert.css]: I guess you can use these colors as variables like we did in levelZero

Comment (srishti-maurya) [reply components/Alert/Alert.css]: That would be convenient for you too, in case any changes are there

**Review by srishti-maurya — APPROVED** [review]

**Review by Amandeep-sinha — APPROVED** [review]

**Pull request merged**